src/controllers/AuthLocalController.js

```diff
@@ -18,27 +18,23 @@ export default class AuthLocalController {
     const {
       firstName, lastName, username, email
     } = req.body;
-    try {
-      const isUser = await helper.isUser({ email });
-      if (isUser) {
-        return res.status(status.EXIST).send({ error: 'Sorry, this account already exists' });
-      }
-      const newUser = await User.create({
-        firstName,
-        lastName,
-        username,
-        email,
-        password: helper.password.hash(req.body.password)
+    const isUser = await helper.isUser({ email });
```

Comment:
why did you remove the try catch block

Reply:
Hello @e-liyai, this file was changed after **rebasing** from **develop**. There were errors in tests, I asked help from my colleague @sengayire who worked on **Authentication** and **User management** to fix this for me. Now, errors are being caught from middlewares and helpers.